core/packages.go

```diff
@@ -686,6 +692,26 @@ func (p *PackageManager) ExistsInRepo(pkg string) error {
 	return nil
 }
 
+func (p *PackageManager) ExistsOnSystem(pkg string) error {
+	PrintVerboseInfo("PackageManager.ExistsOnSystem", "running...")
+
+	PrintVerboseInfo("PackageManager.ExistsInRepo", "checking if package "+pkg+" exists on system: ")
+
+	packageListFile, err := os.ReadFile("/var/lib/dpkg/status")
+	if err != nil {
+		PrintVerboseErr("PackageManager.ExistsOnSystem", 0, err)
+		return err
```

Comment:
I think this should return nil. That way, this functionality won't break on non-debian systems, only throw errors.